### PR TITLE
Less verbosity + code cleanup

### PR DESCRIPTION
**Pull request by RDIL**
Some minor tweaks.

- Make some messages debug only
- Improved the readability of some comments (and typo fix)
- Sync quote style in `build.gradle`
- Fixed the editorconfig

Signed-off-by: Reece Dunham <me@rdil.rocks>

### CODE REVIEW

File: src/main/java/net/fabricmc/loom/LoomGradleExtension.java [hunk 1]
```diff
@@ -58,6 +58,7 @@
 	public String loaderLaunchMethod;
 	public boolean remapMod = true;
 	public boolean autoGenIDERuns = true;
+	public boolean verbose = true;
```
Comment (modmuss50) [inline src/main/java/net/fabricmc/loom/LoomGradleExtension.java]: Whats the use of this? Should you just use --info if you want more verbose logging?

We might need to tweak some log outputs to be lower

**Comment by modmuss50** [comment]
Seems fine overall, not a fan of the verbose thing, using `--info` or `--debug` should be used to get more log output

File: src/main/java/net/fabricmc/loom/LoomGradleExtension.java (hunk 1 above)
Comment (RDIL) [reply src/main/java/net/fabricmc/loom/LoomGradleExtension.java]: Yeah, I think that right now loom outputs a bit too much info for what it is doing.

**Comment by RDIL** [comment]
The one thing I do want to silence that I can't find the source to is `Note: Writing refmap to /workspace/Cherry/build/classes/java/main/CherryClient-refmap.json` - it is repeated like 15 times and its super annoying

**Review by modmuss50 — APPROVED** [review]

**Comment by modmuss50** [comment]
Merge conflicts and then this should be good to go

**Comment by RDIL** [comment]
Fixed.

**Comment by RDIL** [comment]
@modmuss50 all good now!

**Comment by modmuss50** [comment]
Fixed conflicts and merged in my IDE. thanks.

**Pull request closed without merging**